src/main/scala/pl/edu/agh/iet/akka_tracing/couchdb/CouchDatabase.scala

```diff
@@ -69,8 +74,16 @@ class CouchDatabase private[couchdb](baseUrl: String,
     }
   }
 
+  def putDoc[T <: Document](doc: T): Future[Unit] = {
```

Comment:
unused?

Reply:
It was meant to be used, but it isn't anymore. I'll remove this.